FWCore/Framework/bin/cmsRun.cpp

```diff
@@ -332,6 +332,7 @@ int main(int argc, char* argv[]) {
       defaultServices.push_back("AdaptorConfig");
       defaultServices.push_back("SiteLocalConfigService");
       defaultServices.push_back("StatisticsSenderService");
+      defaultServices.push_back("CondorStatusService");
```

Comment:
Why make this a default service? Why not require WMAgent to add it to the configurations since this is only needed on grid jobs?